content/docs/multi-tenancy/_index.md

```diff
@@ -0,0 +1,5 @@
++++
```

Comment:
How about putting this section under the [components](https://www.kubeflow.org/docs/components/) section, rather than giving it a place at the top level of the nav? It'd be good to minimize the number of entries in the top level, so people can get an overview of the doc set more easily.